deploy-renku/entrypoint.sh

```diff
@@ -22,12 +28,12 @@ if test -n "$GITLAB_TOKEN" ; then
                         --data "name=${RENKU_RELEASE}" \
                         --data "redirect_uri=https://${RENKU_RELEASE}.dev.renku.ch/auth/realms/Renku/broker/dev-renku/endpoint https://${RENKU_RELEASE}.dev.renku.ch/api/auth/gitlab/token" \
                         --data "scopes=api read_user read_repository read_registry openid")
-  APP_ID=$(echo $gitlab_app | jq -r '.application_id')
-  APP_SECRET=$(echo $gitlab_app | jq -r '.secret')
+  export APP_ID=$(echo $gitlab_app | jq -r '.application_id')
+  export APP_SECRET=$(echo $gitlab_app | jq -r '.secret')
```

Comment:
I am pretty sure we don't need the export here and in front of `APP_ID` and for `RENKU_VALUES_FILE`. The variables will be available within the script even without export. So no need to do it.